### PR TITLE
Hello Rails: Frontend

### PR DESCRIPTION
**Pull request by smunozmo**
# Documentation

All the documentation is on the README file.

# Backend Repository

https://github.com/smunozmo/hello-rails-backend

## Updates:

- Set up Postgres database.
- Created a table with 5 different greetings.
- Created an API endpoint that selects a random greeting.
- Created the App and Greeting components.
- Created the Greeting component that will display a greeting.
- Created a store, an action and a reducer.

## Notes:

✅  - Used  React.js
✅  - Used  ESLint.
✅  - No linters errors.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @smunozmo, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️

- [ ] To verify that the two apps are working together, I changed the API URL in the React app to my localhost where the backend server was running (127.0.0.1) and the react app showed no message in turn. Please look into this.

> NOTE: I did seed the database so there are existing messages.

_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import Greeting from './components/Greeting';
+
+const App = () => (
+  <Provider store={store}>
+    <div>
+      <Greeting />
+    </div>
+  </Provider>
+
+);
```
Comment (mke2111) [inline src/App.js]: - [x] Within the App component, you are required to add the routes leading to the Greeting component with the help of `react-router` package. Add the necessary dependency and use the available methods from `react-router` to set up the route for Greeting.

File: src/components/Greeting.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+// };
+// class Greeting extends React.Component {
+//   render () {
+//     return (
+//       <React.Fragment>
+//         Greeting: {this.props.greeting}
+//       </React.Fragment>
+//     );
+//   }
+// }
```
Comment (mke2111) [inline src/components/Greeting.js]: - [x] Let us remove this chunk of commented-out code from this file. Standard coding practices don't include pieces of commented-out code.

**Comment by Davidosky007** [comment]
## Hi @smunozmo 👋, 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

done-gif


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by smunozmo** [comment]
Thank you! @Davidosky007 @mke2111

**Pull request merged**